rk3328/manual-image-guide.md

```diff
@@ -117,31 +141,63 @@ $ sudo dd if=/dev/zero of=/dev/sdb bs=512 seek=64 count=24447
 
 ```
 
-## Cloning
+## Cloning Oracular server
+
+
+### Download Ubuntu Oracular preinstalled Server:
```

Comment:
Note that this is not a doc change but a content change. We should probably not mix these.